tests/testthat/test-extract_tag.R

```diff
@@ -0,0 +1,31 @@
+context("extract tags")
+rd <- tempfile()
+
+test_that("multiline title tags are handled reasonably", {
+
+  writeLines("\\title{
+This is a nice title starting with a newline
+}", rd)
+
+  rd_file(rd) %>%
```

Comment:
I think you can simplify this by using `rd_text()`. Then you don't need to write to a temporary file